.gitignore

```diff
@@ -2,3 +2,4 @@
 /db/data
 .env
 db/init/zdummy_data.sql
+.env
```

Comment:
```suggestion
```

already part of the gitignore, the default `.env` file in this repo exists for comfort